src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Tab } from '../../react-app-env';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                data-cy="Tab"
+                className={classNames({
+                  'is-active': id === selectedTab.id,
+                })}
+                onFocus={() => selectedTabId !== id && onTabSelected(tab)}
```

Comment:
Improper event used. Now it fires on Tab key click (try it in browser). Use click event.